openpype/hosts/fusion/addon.py

```diff
@@ -1,7 +1,45 @@
 import os
+import re
 from openpype.modules import OpenPypeModule, IHostAddon
+from openpype.lib import Logger
 
 FUSION_HOST_DIR = os.path.dirname(os.path.abspath(__file__))
+FUSION16_PROFILE_VERSIONS = (16, 17, 18)
+FUSION9_PROFILE_VERSION = 9
+
+
+def get_fusion_profile_number(module: str, app_data: str) -> int:
```

Comment:
Please remove Py3 syntax from addon public code.

Reply:
Fusion does support Py2 but the logic implemented for it has been Py3-focused (e.g. f-strings) so has been Py3 only. Does that still make this problematic?

Reply:
But this is public api of fusion addon (imported anywhere when `ModulesManager` is created).

Reply:
I see - thanks!